guides/gatherers.md

```diff
@@ -0,0 +1,69 @@
+# Gatherers
+
+- [Introduction](#introduction)
+- [Available Gatherers](#available-gatherers)
+    - [Corosync](#corosync)
+
+## Introduction
+
+Gatherers can be thought of as functions:
+- they have a name
+- they accept argument(s)
+- they return a value, the gathered [Fact](./specification.md#facts)
+
+Facts Gathering process in a nutshell 
+```
+fact = gatherer(argument)
+```
+
+## Available Gatherers
+
+Here's a collection of build-in gatherers, with information about how to use them.
+
+| name                          | implementation          
+| ----------------------------- | ------------------------------------------
+| [`corosync.conf`](#corosync)  | [trento-project/agent/../gatherers/corosyncconf.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf.go)
+
+### Corosync
+
+**Name:** `corosync.conf`
+
+This gatherer allows accessing information contained in `/etc/corosync/corosync.conf`
+
+```
+corosync_related_fact = corosync.conf(some.argument)
+```
+
+Accepted arguments
```

Comment:
Those looks like examples to me. They are more options (way more)

Reply:
Changing to `Sample arguments`, anyway there are links to the implementation and tests.
If we want to add more, we can iterate over.